src/librustc_mir/transform/inline.rs

```diff
@@ -808,11 +808,19 @@ impl<'a, 'tcx> MutVisitor<'tcx> for Integrator<'a, 'tcx> {
             }
             TerminatorKind::Abort => { }
             TerminatorKind::Unreachable => { }
-            TerminatorKind::FalseEdges { ref mut real_target, ref mut imaginary_targets } => {
+            TerminatorKind::FalseEdges { ref mut real_target, ref mut imaginary_targets,
+                                         ref mut cleanup } => {
```

Comment:
This is pre-existing, but I wonder if we should just assert that `FalseEdges` is not encountered when doing inlining. I would think we would strip it out and replace it with a plain `Goto` before we begin optimizing. But probably best to leave that for a separate PR.